go/common/enclave.go

```diff
@@ -110,7 +110,6 @@ type Enclave interface {
 
 // BlockSubmissionResponse is the response sent from the enclave back to the node after ingesting a block
 type BlockSubmissionResponse struct {
-	UpdatedHeadRollup       bool                      // Whether the block contained a new head rollup for the canonical rollup chain.
```

Comment:
I got rid of this field. It made removing `HeadsAfterL1Blocks` much easier, and it's simple enough on the host side to check that `IngestedRollupHeader` isn't nil.